src/clr-angular/wizard/_wizard.clarity.scss

```diff
@@ -56,7 +74,7 @@
 
     .modal-body {
       flex: 1 1 auto;
-      color: $clr-wizard-main-textColor;
+      @include css-var(color, clr-wizard-main-text-color, $clr-wizard-main-textColor, $clr-use-custom-properties);
```

Comment:
With the 1:1 matching to help migrations this one is different. I'd say the `-` convention is what we want but for 3.0 I guess it should be `--clr-wizard-main-textColor`?

Reply:
It should be. I'll fix that.